src/main/java/blob/Storage.java

```diff
@@ -0,0 +1,95 @@
+package blob;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Responsible for interfacing with the database (.csv file).
+ * Constructor is to be called with a string representation of the desired file path to store the database as its argument.
+ */
+public class Storage {
+    private File f;
```

Comment:
Maybe can make the name become more explainable to what it do, like f become file